src/DynamoCore/Graph/Workspaces/HomeWorkspaceModel.cs

```diff
@@ -32,6 +32,12 @@ public class HomeWorkspaceModel : WorkspaceModel
         private readonly bool verboseLogging;
         private bool graphExecuted;
 
+        // Event to handle closing of the extension when the workspace is closed. 
+        internal static event Action<String> CloseExtension;
```

Comment:
this seems like a weird name for an event:
```
Do name events with a verb or a verb phrase.

Do give event names a concept of before and after, using the present and past tense. For example, a close event that is raised before a window is closed would be called Closing and one that is raised after the window is closed would be called Closed.

Do not use Before or After prefixes or suffixes to indicate pre and post events.

Do name event handlers (delegates used as types of events) with the EventHandler suffix.

Do use two parameters named sender and e in event handler signatures.

The sender parameter should be of type Object, and the e parameter should be an instance of or inherit from EventArgs.

Do name event argument classes with the EventArgs suffix.
```

Reply:
Yes, I did not give much thought on this naming yet. I will change this accordingly, just waiting for the tests to see if any other changes are needed.